spark/dl/src/main/scala/com/intel/analytics/bigdl/transform/vision/image/MTImageFeatureToBatch.scala

```diff
@@ -127,3 +118,38 @@ private class PreFetch extends Transformer[ImageFeature, ImageFeature] {
     }
   }
 }
+
+/**
+ * A transformer pipeline wrapper to create Minibatch in multiple threads
+ * @param width final image width
+ * @param height final image height
+ * @param totalBatchSize global batch size
+ * @param transformer pipeline for pre-processing
+ * @param toRGB  if converted to RGB, default format is BGR
+ */
+class MTImageFeatureToBatch private[bigdl](width: Int, height: Int,
```

Comment:
`MT` means `Multi threading`, I'd like to have the abstract class to have this name,

Reply:
and name it to another. To keep backward compatibility, Probably we need to have `MTImageFeatureToBatch ` object to instantiate different implementation with options